drivers/sub8_videoray_m5_thruster/launch/thruster_driver.launch

```diff
@@ -1,4 +1,4 @@
 <launch>
     <node pkg="sub8_videoray_m5_thruster" type="thruster_driver.py" name="thruster_driver"
-        args="--calibration_path=$(find sub8_videoray_m5_thruster)/config/calibration.json" />
+        args="--calibration_path=$(find sub8_videoray_m5_thruster)/config/json_nez_prop.json" />
```

Comment:
I guess, this doesn't matter as of now because we need to address issue #267 before being able to run the sub in the current configuration (both stock and Jason's).